locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -2742,14 +2743,53 @@ protected LoginCredentials waitForWinRmAvailable(final ComputeService computeSer
             Callable<Boolean> checker = new Callable<Boolean>() {
                 public Boolean call() {
                     for (Map.Entry<WinRmMachineLocation, LoginCredentials> entry : machinesToTry.entrySet()) {
-                        WinRmMachineLocation machine = entry.getKey();
-                        WinRmToolResponse response = machine.executeCommand(
-                                ImmutableMap.of(WinRmTool.PROP_EXEC_TRIES.getName(), 1),
-                                ImmutableList.of("echo testing"));
-                        boolean success = (response.getStatusCode() == 0);
-                        if (success) {
-                            credsSuccessful.set(entry.getValue());
-                            return true;
+                        try {
+                            final WinRmMachineLocation machine = entry.getKey();
+                            WinRmToolResponse response = machine.executeCommand(
+                                    ImmutableMap.of(WinRmTool.PROP_EXEC_TRIES.getName(), 1),
+                                    ImmutableList.of("echo testing"));
+                            boolean success = (response.getStatusCode() == 0);
+                            if (success) {
+                                credsSuccessful.set(entry.getValue());
+//                                final String pendingRebootStatusScript = "C:\\Get-PendingReboot.ps1";
+//                                ResourceUtils resourceUtils = new ResourceUtils(this);
+//                                int statusCode = machine.copyTo(resourceUtils.getResourceFromUrl("classpath://org/apache/brooklyn/location/winrm/Get-PendingReboot.ps1"), pendingRebootStatusScript);
```

Comment:
I don't see `/org/apache/brooklyn/location/winrm/Get-PendingReboot.ps1` anywhere.

I'd prefer if there was a block comment somewhere that said about the alternative approach of `Get-PendingReboot.ps1`, including a link to https://gallery.technet.microsoft.com/scriptcenter/Get-PendingReboot-Query-bdb79542, and why we're not using it. I really don't like commented out code like this. People can't tell what it's for or why you'd uncomment it without digging around in history and googling.